### PR TITLE
config: angular-schema-form support for ui-select

### PR DESCRIPTION
**Pull request by nikofil**
Signed-off-by: Nikos Filippakis <nikolaos.filippakis@cern.ch>

### CODE REVIEW

**Comment by drjova** [comment]
could you please check the failing tests? :D

File: invenio_deposit/config.py [hunk 1]
```diff
@@ -84,6 +84,24 @@
     ),
 )
 
+DEPOSIT_FORM_TEMPLATES = dict(
+    default='/static/node_modules/invenio-records-js/dist/'
```
Comment (jirikuncar) [inline invenio_deposit/config.py]: Shall we specify only the filename and use `'static'` endpoint to generate full URL?

**Comment by jirikuncar** [comment]
Please re-format commit message to follow: http://invenio.readthedocs.io/en/latest/technology/git.html#r2-remarks-on-commit-log-messages

- No verbs on first line;
- Finish sentences with a dot;
- Check length of lines.

**Comment by nikofil** [comment]
Superseded by https://github.com/inveniosoftware/invenio-deposit/pull/94

**Pull request closed without merging**